app/src/test/java/org/mozilla/fenix/settings/address/controller/DefaultAddressEditorControllerTest.kt

```diff
@@ -73,4 +76,18 @@ class DefaultAddressEditorControllerTest {
             navController.popBackStack()
         }
     }
+
+    @Test
+    fun `GIVEN an existing address record WHEN save address is called THEN update the address record to storage`() = runBlockingTest {
```

Comment:
```suggestion
    fun `GIVEN an existing address record WHEN save address is called THEN update the existing address record in storage`() = runBlockingTest {
```

Reply:
Repeating `existing` would be redundant, we already have the adjective in the `GIVEN` part of the test name. 
"to" and "in" are equivalent in this context. You also used "to storage" in the existing `GIVEN a new address record WHEN save address is called THEN save the new address record to storage` test.